### PR TITLE
 fix(validation): add validation for static, fix multiple choice

### PR DESCRIPTION
**Pull request by czosel**
Depends on #225

### CODE REVIEW

File: addon/components/cf-field/input/table.js [hunk 1]
```diff
@@ -107,6 +107,8 @@ export default Component.extend(ComponentQueryManager, {
 
       this.set("showModal", false);
     } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
```
Comment (anehx) [inline addon/components/cf-field/input/table.js]: Leftover I guess?

File: addon/lib/field.js [hunk 2]
```diff
@@ -234,9 +234,17 @@ export default EmberObject.extend(Evented, {
...
    */
   validate: task(function*() {
+    const specificValidation = this.get(`_validate${this.question.__typename}`);
+    if (!specificValidation) {
+      // eslint-disable-next-line no-console
+      console.error(
+        "Missing validation function for",
+        this.question.__typename
+      );
+    }
```
Comment (anehx) [inline addon/lib/field.js]: Use `assert` from `@ember/debug` for those kind of logs / errors

File: addon/components/cf-field/input/table.js (hunk 1 above)
Comment (czosel) [reply addon/components/cf-field/input/table.js]: No, this is actually intended to avoid swallowing exceptions.

File: addon/lib/field.js (hunk 2 above)
Comment (czosel) [reply addon/lib/field.js]: Fixed. Still have to figure out why the test is failing ...

Comment (czosel) [reply addon/lib/field.js]: Found it, was a curly brace ...

File: addon/components/cf-field/input/table.js (hunk 1 above)
Comment (anehx) [reply addon/components/cf-field/input/table.js]: We used this for debugging the `Array.flat`-IE11-affair..

Comment (fkm) [reply addon/components/cf-field/input/table.js]: Let's keep the error logs until we have fully deployed ember-caluma in all our (main) projects and have more time to harden and clean-up the code.

Comment (czosel) [reply addon/components/cf-field/input/table.js]: @anehx True, but I still like to have those statements around. Lets discuss in person (ETA in office 30min :wink: )

**Pull request merged**